### PR TITLE
README development improvements

### PR DESCRIPTION
**Pull request by NiallRees**
A step-by-step guide. Hopefully this will improve the frequency of contributions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,28 @@ In-depth docs are available [on the Terraform registry](https://registry.terrafo
...
+If you're new to Go:
+
+1. Install Go `brew install golang`
+2. Make a Go development directory wherever you like `mkdir go_projects`
+3. Add the following config to your profile
+```
+export GOPATH=$HOME/../go_projects # edit with your go_projects dir
+export GOROOT=/usr/local/opt/go/libexec
+export PATH=$PATH:$GOPATH/bin
+export PATH=$PATH:$GOROOT/bin
```
Comment (edulop91) [inline README.md:42]: Are these still necessary with Go 1.13+?

Comment (NiallRees) [reply README.md:42]: I literally just started with Go so you'll know more than me! I can try on a fresh environment without

Comment (NiallRees) [reply README.md:42]: Answer is it depends how you install. If you install Go with the package installer from [here](https://golang.org/doc/install) then the required directories are added to PATH. If you install with brew, you need to do it yourself.

Comment (NiallRees) [reply README.md]: You're right, GOROOT no longer required. I've tidied it up and also added a missing step for `goimport`.

**Pull request merged**